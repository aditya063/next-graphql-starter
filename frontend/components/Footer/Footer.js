import styled from 'styled-components';

const StyledFooter = styled.footer`
  text-align: center;
`;

class Footer extends React.PureComponent {
  render() {
    return (
      <StyledFooter>
        <p>Footer</p>
      </StyledFooter>
    );
  }
}

export default Footer;
