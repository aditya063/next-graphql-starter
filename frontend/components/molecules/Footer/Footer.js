// import * as sc from './Footer.style';

const Footer = () => (
  <footer>
    <p>Footer</p>
  </footer>
);

export default Footer;
