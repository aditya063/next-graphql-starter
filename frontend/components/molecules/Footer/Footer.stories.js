import Footer from './Footer';

export default { title: 'Molecules', component: Footer };

export const footer = () => <Footer />;
