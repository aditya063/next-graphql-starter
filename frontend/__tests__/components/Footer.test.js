import { shallow } from 'enzyme';
import Footer from '../../components/Footer/Footer';

describe('Footer', () => {
  let mockProps;
  let wrapper;

  beforeEach(() => {
    jest.resetAllMocks();
    mockProps = {};
    wrapper = shallow(<Footer {...mockProps} />, {
      disableLifecycleMethods: true
    });
  });

  it('matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
