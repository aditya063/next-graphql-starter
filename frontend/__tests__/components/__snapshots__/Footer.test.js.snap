// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches snapshot 1`] = `
<Footer__StyledFooter>
  <p>
    Footer
  </p>
</Footer__StyledFooter>
`;
